package.json

```diff
@@ -12,14 +12,15 @@
     "build": "webpack --config ./frontend_build/src/webpack.config.prod.js",
     "test-build": "yarn run lint-js && webpack --config ./frontend_build/src/webpack.config.prod.js --bail",
     "watch": "node ./frontend_build/src/webpackdevserver.js",
+    "django-devserver": "kolibri --debug manage devserver --settings=kolibri.deployment.default.settings.dev ",
+    "devserver": "./node_modules/.bin/npm-run-all --parallel django-devserver watch",
```

Comment:
you can omit the path to `npm-run-all`. in the context of package.json scripts, `.bin` is considered as part of the "path"

Reply:
I have seen cases where this has not worked consistently, which is why the fuller path is here.